sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -43,8 +44,12 @@ def _return_response(pipeline_response, deserialized, response_headers):
 
 
 class ContainerRegistryClient(ContainerRegistryBaseClient):
-    def __init__(self, endpoint, credential=None, **kwargs):
-        # type: (str, Optional[TokenCredential], **Any) -> None
+    def __init__(
+        self,
+        endpoint: str,
+        credential: "Optional[TokenCredential]"=None,
```

Comment:
```suggestion
        credential: Optional["TokenCredential"] = None,
```
I think we only need quotes around the credential type.

Reply:
Updated.